tilelang/cache/kernel_cache.py

```
@@ -583,19 +586,29 @@ def _load_kernel_from_disk(
+                host_kernel_source=CachedTextSource(path=host_kernel_path),
+                device_kernel_source=CachedTextSource(path=device_kernel_path),
+                kernel_lib_path=kernel_lib_path,
+                kernel_params=kernel_params,
+                target=target,
+                target_host=target_host,
+                out_idx=out_idx,
+                execution_backend=execution_backend,
+                pass_configs=pass_configs,
+                compile_flags=compile_flags,
+            )
+        except Exception as err:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)